master/src/main/scala/AuxFunction.scala

```diff
@@ -0,0 +1,39 @@
+package kr.ac.postech.paranode.master
+
+import kr.ac.postech.paranode.core.WorkerMetadata
+
+import java.nio.file.Files
+import java.nio.file.Paths
+import scala.collection.mutable.ListBuffer
+import scala.io.Source
+
+object AuxFunction {
+
+  def getWorkerDetails(): List[WorkerMetadata] = {
+
+    val dirPath = Paths.get("worker_register")
```

Comment:
Transfer worker information via file is not a good idea. It's **unpure** action.

Reply:
https://github.com/betarixm/434project/pull/71/commits/b5db92ab7c09755f9de355b1f7906e129cf55542
https://github.com/betarixm/434project/pull/71/commits/5e04944846cc46b9880281e3de16844b7da49177

Reply:
I changed file related thing into mutable variable:ListBuffer.
I think the emerging state is inevitable.